modes.go

```diff
@@ -393,11 +393,6 @@ func DoReadsFromTable(table string, session *gocql.Session, threadResult *result
 		selectFields = "pk, ck, c1, c2, c3, c4, c5"
```

Comment:
With this case the orderBy won't work will it?

Reply:
It does

Reply:
Right. ck is first here too.